gratipay/testing/browser.py

```diff
@@ -82,10 +91,51 @@ def has_element(self, selector, timeout=None):
         """
         return self.is_element_present_by_css(selector, timeout)
 
+    def wait_to_disappear(self, selector, timeout=2):
+        """Wait up to ``timeout`` seconds for element specified by ``selector``
+        to disappear, returning ``None``.
+        """
+        end_time = time.time() + timeout
+        while time.time() < end_time:
+            if not self.has_element(selector):
+                return
+        raise NeverLeft()
```

Comment:
It looks in 3c1e47ecd62124ba90b5041289b78340bb8d7ce0 like passing in the selector is sufficient to get it out again in a traceback.